packages/eas-cli/src/submissions/android/AndroidSubmitCommand.ts

```diff
@@ -65,13 +65,13 @@ class AndroidSubmitCommand {
 
   private resolveAndroidPackageSource(): Result<AndroidPackageSource> {
     let androidPackage: string | undefined;
-    if (this.ctx.commandFlags.androidPackage) {
-      androidPackage = this.ctx.commandFlags.androidPackage;
-    }
     const { exp } = getConfig(this.ctx.projectDir, { skipSDKVersionRequirement: true });
     if (exp.android?.package) {
       androidPackage = exp.android.package;
     }
+    if (this.ctx.commandFlags.androidPackage) {
+      androidPackage = this.ctx.commandFlags.androidPackage;
+    }
```

Comment:
Alternatively, you can use `else-if` - IMO it's more readable and straightforward - then the logic flow is not "inverted" - it goes straight up - down line by line
```suggestion
    const { exp } = getConfig(this.ctx.projectDir, { skipSDKVersionRequirement: true });
    if (this.ctx.commandFlags.androidPackage) {
      androidPackage = this.ctx.commandFlags.androidPackage;
    } else if (exp.android?.package) {
      androidPackage = exp.android.package;
    }
```